server/auth.js

```diff
@@ -0,0 +1,102 @@
+const auth = require('express').Router();
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+
+const User = require('../db').model('users');
+
+passport.serializeUser((user, done) => {
+  done(null, user.id);
+});
+
+passport.deserializeUser(
+  (id, done) => {
+    User.findById(id)
+      .then(user => {
+        done(null, user);
+      })
+      .catch(err => {
+        done(err);
+      });
+  }
+);
+
+// Local login
+auth.post('/local/login', (req, res, next) => {
+  passport.authenticate('local', {
+    successRedirect: '/'
```

Comment:
Ditto on the earlier comment. I think that we should redirect to that URL that the user entered before getting redirected to the login page.